schema/annotation-source.yaml

```diff
@@ -25,22 +25,23 @@ $defs:
         type: string
         description: >-
           A free-text description of the InformationEntity.
-      strength:
+      confidence_level:
```

Comment:
The definition of the 'strength' attribute you had created here sounded specific to a Statement.  I proposed below to define a strength property in the Statement object itself ('evidence_strength) . . .this is consistent with the VA Core model.  

If you do want a property to describe strength/confidence in a InformationEntity, the VA core provides a general `InformationEntity.confidence_level` attribute that can be applied to any type of InformationEntity.  But there may not be a need for this, if we define a 'strength' attribute on Statement directly.

Reply:
I'm okay with the term `strength` but I really think that this term is conveying the `strength of the assertion`. `evidence_strength` conveys a sense of how strong the evidence is, which may not be the same thing as the how strongly the asserter supports or refutes. It really is more about the strength of certainty IMO.  
I would prefer to decouple this attribute from the evidence - even though there is some indirect relationship.

Reply:
I may be confused here. If we are calling this `confidence_level` then I think that fits. As long as it is the confidence level in the direction of the assertion relative to the proposition (i.e. support, refute or uncertain).  Of course uncertain would always be the the confidence level of an uncertain assertion of a proposition.

Reply:
The main issue I wanted to focus on here Larry is that this `strength` attribute in the metakg-cvc schema is defined as if it were specific to a `Statement`, but it hangs from the `InformationEntity` class  here (which is not immediately apparent from how GH displays things ).

I simply propose to move attributes meant to capture Statement-specific strength information down to the `Statement` class. And if there is a use case for wanting to describe confidence/quality/strength of any given `InformationEntity` -  we can use the `InformationEntity.confidence_level` property  defined in the VA Core IM for this purpose. This attribute would be inherited by descendant classes including Statement, where it could be used to capture an agent's confidence in the truth of a Proposition (but I think this is turnout out to be something different than the notion of 'strength')

Re: this related issue of what to call and how to define this 'strength' attribute once moved to the `Statement` class, and how it relates to the notion of 'evidence strength' vs 'confidence_level' - I'll treat this separately in replies below.